src/Creator.java

```diff
@@ -24,7 +24,12 @@ public void createWorkOrders() throws IOException {
 
         // write as json files
 
+        String fileName = order.getId() + ".json";
+
         ObjectMapper mapper = new ObjectMapper();
+
+        mapper.writeValue(new File(fileName), order);
```

Comment:
This can throw an IOException so it should be wrapped in a try/catch. I would just have the catch print a quick message that there was an error and then print out the stack trace to kill the program.